pkg/sqs/sqs.go

```diff
@@ -88,14 +87,7 @@ func (s *AWSSqsSource) Read(_ context.Context, readRequest sourcesdk.ReadRequest
 
 	ctx, cancel := context.WithTimeout(context.Background(), readRequest.TimeOut())
```

Comment:
I don't think we need this and line 105. As long as the messages have been read from SQS, we should push them into the channel.

Reply:
+1